src/core/Analytics.ts

```
@@ -4,12 +4,20 @@ import Constants from '@covid/utils/Constants';
 import * as Amplitude from 'expo-analytics-amplitude';
 
 import { TScreenName } from './Coordinator';
+import { firebaseAnalytics } from './firebase/analytics';
+
+const fbAnalytics = firebaseAnalytics();
+try {
+  fbAnalytics.isInitialized();
+} catch (e) {
+  throw new Error('firebase analytics not initialized');
```

Do we really want to throw an error here?

---

🤔 gd question, the analytics is not critical so I think we can not throw an error and instead have a sentry alert maybe? or some kind of logging - will change this